Tests/TestData/optmizelyConfig/optimizely_config_datafile.json

```diff
@@ -243,10 +318,30 @@
             "id": "12115595439"
         }
     ],
-    "attributes": [],
+    "typedAudiences": [
+      {"id": "20348452263", "conditions": ["and", ["or", ["or", {"value": 18, "type": "custom_attribute", "name": "age", "match": "gt"}]]], "name": "adult"},
+      {"id": "20348352569", "conditions": ["and", ["or", ["or", {"value": 18, "type": "custom_attribute", "name": "age", "match": "lt"}]]], "name": "kid"}
+    ],
+    "audiences": [
```

Comment:
Can we have 1 or 2 unique audiences in both `typedAudiences` and `audiences` to verify if they are being merged correctly?